File: src/main/java/com/musinsa/subject/model/product/ProductResponse.java
package com.musinsa.subject.model.product;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ProductResponse {

    @Schema(defaultValue = "0")
    private Long productId;
    @Schema(defaultValue = "0")
    private Long productPrice;

}

File: src/main/kotlin/com/musinsa/subject/model/product/ProductResponse.kt
package com.musinsa.subject.model.product

import io.swagger.v3.oas.annotations.media.Schema

class ProductResponse(
    @get:Schema(defaultValue = "0")
    var productId: Long? = null,

    @get:Schema(defaultValue = "0")
    var productPrice: Long? = null
)
